test/srt/test_chunked_prefill.py

```diff
@@ -30,21 +42,38 @@ def run_mmlu(
             base_url,
             timeout=DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
             other_args=other_args,
+            return_stdout_stderr=True,
         )
 
+        output_lines = []
+        t = threading.Thread(target=read_output, args=(process, output_lines))
+        t.start()
```

Comment:
QQ What is the lifecycle of this thread?
Minor: Would it be better to check this thread after the eval has finished running?

Reply:
We have a `kill_child_process(process.pid)` to clean up all of them. Hopefully, it can be correctly cleaned up.